Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -182,7 +182,19 @@ private void Scrobble(PlexEvent data, SVR_JMMUser user)
         if (episodeType != EpisodeType.Episode ||
             metadata.Index == 0) //metadata.index = 0 when it's something else.
         {
-            return (null, anime); //right now no clean way to detect the episode. I could do by title.
+            var nameMatches = anime
+                .GetAnimeEpisodes().Where(a => a.AniDB_Episode != null)
+                .Where(a => a.EpisodeTypeEnum == episodeType)
+                .Where(a => a.Title.Equals(metadata.Title))
+                .Where(a => a.AniDB_Episode.EpisodeNumber == episodeNumber
+                    || (a.TvDBEpisode?.SeasonNumber == series && a.TvDBEpisode?.EpisodeNumber == episodeNumber)).ToList();
```

Comment:
Not a huge fan of this - the reason is doing this "a or b" could cause erroneous matches.